### DISCUSSION TITLE
Carousel is rendering items 3 times the length of array

### DISCUSSION THREAD

**Discussion by danishgrappus** [vadymshymko/react-simply-carousel#121] [Q&A] [answered]
Attaching a sandbox for full clarity : https://codesandbox.io/s/react-simply-carousel-demo-forked-zspf45?file=/src/index.js
So, if my array has 2 items, it renders 6 divs. If array has 7 items, then 21 divs.
Please help.
**Expected result:**
It should just render the nos of items equal to length of array

**Comment by vadymshymko** [comment, marked as answer]
Hi @danishgrappus 
This is expected correct behavior for the current implementation of infinite scroll.  However, I’m already thinking about how to get away from this in future versions.
